package-lock.json

```diff
@@ -0,0 +1,5 @@
+{
```

Comment:
Hi Lawrie, did you want the package-lock.json file included this time?  Also, it has the previous version number in it.